src/modules/blocks/attendees/container.js

```diff
@@ -8,19 +8,33 @@ import { bindActionCreators, compose } from 'redux';
  * Internal dependencies
  */
 import { withSaveData, withStore } from '@moderntribe/common/hoc';
-import * as actions from '@moderntribe/tickets/data/blocks/attendees/actions';
-import * as selectors from '@moderntribe/tickets/data/blocks/attendees/selectors';
+import * as UIActions from '@moderntribe/tickets/data/blocks/attendees/actions';
+import * as UISelectors from '@moderntribe/tickets/data/blocks/attendees/selectors';
 import Attendees from './template';
 
 /**
  * Module Code
  */
 
 const mapStateToProps = ( state ) => ( {
-	title: selectors.getTitle( state ),
+	title: UISelectors.getTitle( state ),
+	displayTitle: UISelectors.getDisplayTitle( state ),
+	displaySubtitle: UISelectors.getDisplaySubtitle( state ),
 } );
 
-const mapDispatchToProps = ( dispatch ) => bindActionCreators( actions, dispatch );
+const mapDispatchToProps = ( dispatch ) => ( {
+	...bindActionCreators( UIActions, dispatch ),
```

Comment:
Might be a good idea to specify the actions being passed here instead to make it clear which ones are the ones used into the component.

Reply:
I don't understand this one. Can you please give me an example? I assumed I'm already specifying the actions being passed here (at least the ones I've added). Thanks.

Reply:
What @mitogh is talking about is something like https://github.com/moderntribe/event-tickets/blob/1cae571db0aa927d160deaf78d25090c3afe72e7/src/modules/blocks/rsvp/container-header/container.js#L29 that explicitly dispatches the actions. Using `bindActionCreators()` is concise, but it's not declarative and makes code difficult to follow.

Also, it looks like below you've got 2 dispatch props defined, which dispatch `UIActions`. This means you're double-declaring dispatch props, you'll have one called `onSetDisplayTitleChange` and another called `setDisplayTitle` which do the exact same thing (we're just passing unnecessary props). Best to declare your dispatch props declaratively rather than imperatively.

Reply:
Yes, sorry for the confusion.

Instead of spreading all the properties from `bindActionCreators` you specify only the actions you need, like: 

https://github.com/moderntribe/event-tickets/blob/master/src/modules/blocks/rsvp/container.js#L64-L69

So it's clear which actions are passed in the components.

Reply:
Ok, I get it, that will make it more precise in terms of specificity, but I can't get it to work with the title change actions. If you want to enlighten me on how to do that I'll appreciate it.

Reply:
for example you might have:
```js
const mapDispatchToProps = ( dispatch ) => ( {
    setDisplayTitle: ( checked ) => dispatch( actions.setDisplayTitle( checked ) );,
    ...
} );
```

Reply:
Yes, that will work with the set display title, but the block also has an input to edit the text of the title. That's what I mean.

Reply:
Besides, if I do that, the onchange for the toggle stop working as well.

Reply:
I could write it out here, but maybe we should hop on a call to work this out. It might be faster.

Reply:
@paulmskim thanks for the clarification. I've just submitted the changes we chatted about. Please let me know how you see it